pkg/storage/tsdb/config.go

```diff
@@ -490,3 +509,15 @@ func (cfg *BucketIndexConfig) Validate(logger log.Logger) error {
 	}
 	return nil
 }
+
+type ActiveSeriesMetricsConfig struct {
+	Enabled      bool          `yaml:"active_series_metrics_enabled" category:"advanced"`
+	UpdatePeriod time.Duration `yaml:"active_series_metrics_update_period" category:"advanced"`
+	IdleTimeout  time.Duration `yaml:"active_series_metrics_idle_timeout" category:"advanced"`
+}
+
+func (cfg *ActiveSeriesMetricsConfig) RegisterFlags(f *flag.FlagSet) {
+	f.BoolVar(&cfg.Enabled, activeSeriesMetricsEnabledFlag, true, "Enable tracking of active series and export them as metrics.")
+	f.DurationVar(&cfg.UpdatePeriod, "ingester.active-series-metrics-update-period", 1*time.Minute, "How often to update active series metrics.")
+	f.DurationVar(&cfg.IdleTimeout, activeSeriesMetricsIdleTimeoutFlag, 10*time.Minute, "After what time a series is considered to be inactive.")
+}
```

Comment:
Nit: I think this should live in the `activeseries` package and being called just `activeseries.Config`.

Reply:
Done: https://github.com/grafana/mimir/pull/5371/commits/90640d302a14d414ea804b5e6d97d61892e9413d